integration/js/pages/AppPage.js

```diff
@@ -77,9 +81,24 @@ class AppPage {
     await authenticateButton.click();
   }
 
+  async chooseUseWebAudio() {
+    let webAudioFeature = await this.driver.findElement(elements.webAudioFeature);
+    let webAudioFeatureLabel = await this.driver.findElement(elements.webAudioFeatureLabel);
+    if (await webAudioFeature.isSelected()) {
+         console.log('Web Audio is selected');
```

Comment:
Nit: indenting. I can fix this when I come by here again.

Reply:
updated